cmd/thanos/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	logLevel := app.Flag("log.level", "Log filtering level.").
 		Default("info").Enum("error", "warn", "info", "debug")
-	logFormat := app.Flag("log.format", "Log format to use.").
+	logFormat := app.Flag("log.format", "Log format to use, support logfmt or json").
```

Comment:
```suggestion
	logFormat := app.Flag("log.format", "Log format to use. Possible options: logfmt or json").
```

Reply:
We might do this event for the log level